README.md

```diff
@@ -264,6 +265,16 @@ assume([]).is.a('array');
 
 [`instanceof` is a keyword and might cause cross browser issues](#keywords)
 
+#### eitherOfType, oneOfType
+
+Asserts if the given value is one of the acceptable types.
+
+The same caveats regarding `typeof` apply as described in [a, an](#a-an).
+
+```js
+assume([]).is.oneOfType(['array'], ['string']);
```

Comment:
this should be a single array as a single argument rather than 2 arrays correct?